site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -165,18 +165,45 @@ We need a firewall to allow UDP traffic to nodes tagged as `game-server` via por
 next section.
 
 ```bash
-gcloud compute firewall-rules create game-server-firewall \
+gcloud compute firewall-rules create gke-agones-game-server-firewall \
   --allow udp:7000-8000 \
   --target-tags game-server \
   --description "Firewall to allow game server udp traffic"
 ```
 
 ## Creating the cluster
 
-Create a GKE cluster in which you'll install Agones. You can use
+Create a GKE cluster (preferably an Autopilot mode cluster) in which you'll install Agones. You can use
 [GKE Standard mode](#create-a-standard-mode-cluster-for-agones)
 or [GKE Autopilot mode](#create-an-autopilot-mode-cluster-for-agones).
 
+### Create an Autopilot mode cluster for Agones
+
+{{<alert title="Note" color="info">}}
+These installation instructions apply to Agones 1.30+
+{{</alert>}}
```

Comment:
Not your fault but might as well handle it here: We had different instructions in 1.29 so the goal was to call them out. But it's been over a year - I think it's far enough past that we can remove this clause.